.github/workflows/version_handling.yml

```diff
@@ -35,4 +35,4 @@ jobs:
       - name: Check we got the right version (devel/release)
         if: matrix.version == 'devel' || startsWith(matrix.version, 'release')
         run: |
-          [[ "$(~/.local/bin/subctl version)" =~ ^"subctl version: subctl-${{ matrix.version }}"(-[0-9]+-g[a-f0-9]+)?$ ]]
+          [[ "$(~/.local/bin/subctl version)" =~ ^"subctl version: ${{ matrix.version }}"(-[0-9]+-g[a-f0-9]+)?$ ]]
```

Comment:
This fails for _release*_  versions as they are prefixed with _subctl_ (eg _subctl-release-0.8-1-g205505d_). We need to have separate steps for _devel_ and _release*_ versions.

Reply:
Only release-0.8 is prefixed with `subctl`.

```
curl -Ls https://get.submariner.io | VERSION=release-0.12 bash
export PATH=$PATH:~/.local/bin
echo export PATH=\$PATH:~/.local/bin >> ~/.profile
Installing subctl version release-0.12
  OS detected:           linux
  Architecture detected: amd64
  Download URL:          https://github.com/submariner-io/submariner-operator/releases/download/subctl-release-0.12/subctl-release-0.12-linux-amd64.tar.xz

Downloading...
subctl-release-0.12-linux-amd64 has been installed as /home/janki/.local/bin/subctl
This provides subctl version: release-0.12
```